nym-api/nym-api-requests/src/coconut/models.rs

```diff
@@ -135,7 +135,7 @@ pub struct FreePassRequest {
     // secp256k1 key associated with the admin account
```

Comment:
so isn't the inner request actually the same one as for "normal" credential? (minus nonce + signature)

Reply:
Yes